public/app/plugins/datasource/testdata/nodeGraphUtils.ts

```diff
@@ -135,9 +122,10 @@ export function generateRandomNodes(count = 10) {
         continue;
       }
       edgesSet.add(id);
-      edgeFields.id.values.add(`${node.id}--${edge}`);
-      edgeFields.source.values.add(node.id);
-      edgeFields.target.values.add(edge);
+      edgesFrame.fields[0].values.add(`${node.id}--${edge}`);
```

Comment:
Could not work out the typing without using type assertion so ended with an array instead of object here.